### PR TITLE
feat: update to inform about how to validate data

### LINKED ISSUE #16: Update DATA_OVERLAY.md
As a data contributor, I would like to have information in DATA_OVERLAY.md that I can use the custom data upload user functionality in map viewer's data overlay bar to validate my files, so that I know how to verify that it is correct before requesting the addition of new data sources

This readme update covers only for the new data validation.

### PR DESCRIPTION
**Pull request by inghylt**
This issue closes #16

### CODE REVIEW

File: DATA_OVERLAY.md [hunk 1]
```diff
@@ -55,4 +55,6 @@ Inside of a data type folder, there could be one or more data source files, such
 
 When incorporating new data sources, it should be done via a pull request in the [data-files repo](https://github.com/MetabolicAtlas/data-files). The above rules and conventions about data format as well as folder structure should be followed.
 
+To validate the data to be added, please try to load it as custom expression data at the [data overlay functionality](localhost/explore/Human-GEM/map-viewer/golgi_apparatus?dim=2d&panel=1) of Metabolic Atlas. If no errors are reported, the data is formatted correctly.
```
Comment (mihai-sysbio) [inline DATA_OVERLAY.md:58]: The `localhost` link would only work for devs, maybe it's easier if we generalize:

```suggestion
To validate the data to be added, please try to load it as custom expression data at the [data overlay functionality](https://metabolicatlas.org/explore/Human-GEM/map-viewer/golgi_apparatus?dim=2d&panel=1) of Metabolic Atlas. If no errors are reported, the data is formatted correctly.
```

**Review by mihai-sysbio — APPROVED** [review]
Nice and clear!

File: DATA_OVERLAY.md (hunk 1 above)
Comment (inghylt) [reply DATA_OVERLAY.md:58]: Haha wow, thanks for noticing that

**Review by nanjiangshu — APPROVED** [review]
Clear enough!

**Pull request merged**